packages/fabric8-tenant-che-mt/src/main/fabric8/user-exec-rb.yml

```diff
@@ -0,0 +1,10 @@
+metadata:
+  name: user-exec
+roleRef:
+  name: exec
+  namespace: ${PROJECT_NAMESPACE}-che
+subjects:
+- kind: User
+  name: ${PROJECT_USER}
+userNames:
```

Comment:
As far as I remember this field will be ignored since `subject` field is specified. The corresponding OpenShift doc is [here](https://docs.openshift.com/container-platform/3.5/rest_api/openshift_v1.html#v1-rolebinding).
Did you specify both for backward compatibility? Do we have old OpenShift instances which don't support `subjects`?

Reply:
yeah, I believe those were initially added for backward compatibility, so that it would be possible to run the generated template everywhere e.g.  - http://central.maven.org/maven2/io/fabric8/tenant/packages/fabric8-tenant-che-mt/2.0.86/